servicebus/data-plane/azure-servicebus/src/main/java/com/microsoft/azure/servicebus/management/RuleDescriptionSerializer.java

```diff
@@ -213,11 +215,12 @@ static RuleDescription parseFromContent(String xml) throws MessagingEntityNotFou
         DocumentBuilderFactory dbf = DocumentBuilderFactory.newInstance();
         try {
             DocumentBuilder db = dbf.newDocumentBuilder();
-            Document dom = db.parse(new ByteArrayInputStream(xml.getBytes("utf-8")));
+            Document dom = db.parse(new ByteArrayInputStream(xml.getBytes(UTF_8)));
             Element doc = dom.getDocumentElement();
             doc.normalize();
-            if (doc.getTagName() == "entry")
+            if (doc.getTagName() == "entry") {
```

Comment:
>f (doc.getTagName() == "entry") { [](start = 13, length = 33)

Change this to equals.

Reply:
the changes applied in https://github.com/Azure/azure-sdk-for-java/pull/3539/files#diff-a3d16a173fe7aafcc18ca133e9f8a908R219